docs/read_and_write.rst

```diff
@@ -126,6 +127,70 @@ path must be wrapped in backticks.
 
   Field names containing periods (``.``) are not supported.
 
+Literals for dates, timestamps, and decimals can be written by writing the string
+value after the type name. For example
+
+  .. code-block:: SQL
+
+    date_col = date '2021-01-01'
+    and timestamp_col = timestamp '2021-01-01 00:00:00'
+    and decimal_col = decimal(8,3) '1.000'
+
+For timestamp columns, the precision can be specified as a number in the type
+parameter. Microsecond precision (6) is the default.
+
+.. list-table::
+    :widths: 30 40
+    :header-rows: 1
+
+    * - SQL
+      - Time unit
+    * - ``timestamp(0)``
+      - Seconds
+    * - ``timestamp(3)``
```

Comment:
is the precision a limitation by the parser?  Feel that `timestamp(s)`, `timestamp(ms)` are easier to use. Curious how much effort to make it happen?

https://arrow.apache.org/docs/python/generated/pyarrow.timestamp.html

Reply:
Yeah that's unfortunately something that's not built in; it only allows integers.

Reply:
Ok, if this cannot be done in a few hours, let's just 🚢 it for now.